Makefile

```diff
@@ -155,3 +155,10 @@ env-sysbench-run:
 		--db-driver=pgsql --pgsql-host=postgres --pgsql-user=pmm-agent --pgsql-password=pmm-agent-password --pgsql-db=pmm-agent \
 		--threads=4 --time=0 --rate=10 --report-interval=10 --percentile=99 \
 		--tables=1 --scale=10  --use_fk=0 --enable_purge=yes run
+
+check-all: check        ## Run golang ci linter to check new changes from master.
```

Comment:
That should follow `check` target, and doc comments should be indented with the same number of spaces.
Basically, port https://github.com/percona/pmm-managed/commit/cf59a2d57f6c2b966160c9a959a4a3fdd780cafb. Please do the same for other PRs.